docs/accessControl.md

```diff
@@ -164,10 +164,15 @@ To block students from viewing closed assessment details, set `"showClosedAssess
         "timeLimitMin": 50,
         "showClosedAssessment": false,
         "credit": 100
+    },
+    {
+        "showClosedAssessment": false
     }
 ]
 ```
 
+Note that the `showClosedAssessment` statement is listed twice above, once while the assessment is in progress (`startDate` to `endDate`) and once in a second rule that will take effect after the `endDate`.
```

Comment:
May want to include a comment that new submissions are allowed after this point if this new rule is added, so if they don't want that they should set `"active": false` (maybe a link to the corresponding section).

Reply:
But won't the assessments be closed by this point, so that they won't even be viewable by the student?

I can certainly add `"active": false` to the final rule just to be on the safe side, if you think it would help?

Reply:
> But won't the assessments be closed by this point, so that they won't even be viewable by the student?

Not if they didn't start.

Reply:
Good point. Added in 90d0092d3d1cab14c60e1ebc60380658562587a9